CodeSuggestion has new field in latest nightly

https://github.com/rust-lang/rust/pull/43096 introduced `show_code_when_inline` to `rustc_errors::CodeSuggestion` so clippy_lints (and perhaps other stuff) fail to compile on the latest Rust nightly.

```
   Compiling clippy_lints v0.0.144
error[E0063]: missing field `show_code_when_inline` in initializer of `rustc_errors::CodeSuggestion`
   --> /home/admin/.cargo/registry/src/github.com-1ecc6299db9ec823/clippy_lints-0.0.144/src/utils/mod.rs:577:16
    |
577 |     let sugg = rustc_errors::CodeSuggestion {
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `show_code_when_inline`

error: aborting due to previous error

error: Could not compile `clippy_lints`.
```

--- a/clippy_lints/src/utils/mod.rs
+++ b/clippy_lints/src/utils/mod.rs
@@ -584,6 +584,7 @@ pub fn multispan_sugg(db: &mut DiagnosticBuilder, help_msg: String, sugg: Vec<(S
             })
             .collect(),
         msg: help_msg,
+        show_code_when_inline: true,
     };
     db.suggestions.push(sugg);
 }

